incubator/virtualcluster/pkg/syncer/conversion/mutate.go

```diff
@@ -418,6 +419,28 @@ func (s *serviceMutator) Mutate(vService *v1.Service) {
 	}
 }
 
+type IngressMutateInterface interface {
```

Comment:
Since we don't call mutate anymore in dws.go, can we remove all the changes in this file?

Reply:
Yes, good call.

Reply:
Done.